guides/common/modules/proc_creating-compute-profiles.adoc

```diff
@@ -2,6 +2,7 @@
 = Creating compute profiles
 
 You can use compute profiles to predefine virtual machine hardware details such as CPUs, memory, and storage.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-compute-profiles_{context}[]. +
```

Comment:
```suggestion

To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-compute-profiles_{context}[].

```
Most of the other occurrences of this sentence in the guide use a separate paragraph. Also, the plus sign at the end seems like it should not be there.